src/App.tsx

```diff
@@ -15,36 +15,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET = 'alphabet';
+
+type SortField = 'length' | 'alphabet' | string;
```

Comment:
This type basically not very usefull since I can set any string.

let's create enum here for instance 

```typescript
enum SortType {
  LENGTH:  'length';
  ALPHABET: 'alphabet'
  NONE: 'none' // this will be our default value
```